zipline/data/minute_bars.py

```diff
@@ -877,7 +911,9 @@ def unadjusted_window(self, fields, start_dt, end_dt, sids):
                             excl_start - start_idx:excl_stop - start_idx + 1]
                         values = np.delete(values, excl_slice)
                 where = values != 0
-                out[i, where] = values[where]
+                # first slice down to len(where) because we might not have
+                # written data for all the minutes requested
+                out[:len(where), i][where] = values[where]
```

Comment:
There are two changes here. The first is that we are transposing the output to be more consistent with the other readers. The second is that we are adding some extra slicing to make sure the indicies are aligned when write. This was causing warnings.